.circleci/test-deploy.yml

```diff
@@ -1,38 +1,47 @@
 version: 2.1
 orbs:
-  <orb-name>: <namespace>/<orb-name>@dev:<<pipeline.git.revision>>
+  testlio: testlio/testlio@dev:<<pipeline.git.revision>>
   orb-tools: circleci/orb-tools@11.1
 
 filters: &filters
   tags:
     only: /.*/
 
 jobs:
-  # Create a job to test the commands of your orbs.
-  # You may want to add additional validation steps to ensure the commands are working as expected.
-  command-tests:
+  unzip-sample-test-project:
     docker:
-      - image: cimg/base:current
+      - image: cimg/node:lts
     steps:
       - checkout
-      # Run your orb's commands to validate them.
-      - <orb-name>/greet
+      - run:
+          name: Unzip Sample Test Project
+          command: "unzip testlio-platform-automation-main.zip"
+      - persist_to_workspace:
+          root: ./testlio-platform-automation-main
+          paths:
+            - ./*
 workflows:
   test-deploy:
     jobs:
-      # Make sure to include "filters: *filters" in every test job you want to run as part of your deployment.
-      - command-tests:
+      - unzip-sample-test-project:
```

Comment:
Why do we need to unzip the sample test project? Testlio CLI only needs the zip file.